packages/block-library/src/columns/edit.js

```diff
@@ -135,9 +135,11 @@ export function ColumnsEdit( {
 		[ `are-vertically-aligned-${ verticalAlignment }` ]: verticalAlignment,
 	} );
 
+	const showTemplateSelector = ( count === 0 && ! forceUseTemplate ) || ! template;
+
 	return (
 		<>
-			{ template && (
+			{ ! showTemplateSelector && (
```

Comment:
Alternatively, I wonder what it is about choosing columns that causes the issue in the first place. If someone wanted to skip the layout selection and just start setting columns (assuming of an equal width), why prevent them?

Reply:
yeah I'm not sure what. I decided to remove the selector not only because of the bug because based on the previous check here. I assumed that removing the inspector controls was an intentional behavior.